swaymsg/main.c

```diff
@@ -440,6 +440,8 @@ int main(int argc, char **argv) {
 		type = IPC_GET_VERSION;
 	} else if (strcasecmp(cmdtype, "get_binding_modes") == 0) {
 		type = IPC_GET_BINDING_MODES;
+	} else if (strcasecmp(cmdtype, "get_binding_state") == 0) {
```

Comment:
Can you please also add this to `swaymsg.1.scd`?

Reply:
Updated swaymsg.1.scd and rebased